.github/CONTRIBUTING.md

```diff
@@ -31,7 +31,15 @@ Please post code and output as text ([using proper markup](https://guides.github
 5. Ensure the code complies with our coding guidelines (see below).
 6. Send that pull request!
 
-Please make sure you have [set up your username and email address](https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup) for use with Git. Strings such as `silly nick name <root@localhost>` look really stupid in the commit history of a project.
+## Specific set up for documentation application
+
+1. You should have installed `node`(_we recommend using [nvm](https://github.com/nvm-sh/nvm)_) and `yarn`.
```

Comment:
Given how this sentence is written, it gives the impression that using yarn is a recommendation and not a requirement. In reality, since we have locked dependencies (there's a yarn.lock at the root of the repository), yarn is a requirement.

Reply:
You're right,I will write this sentence in another way